apps/meteor/package.json

```diff
@@ -272,7 +272,7 @@
 		"@rocket.chat/ui-client": "workspace:^",
 		"@rocket.chat/ui-composer": "workspace:^",
 		"@rocket.chat/ui-contexts": "workspace:^",
-		"@rocket.chat/ui-kit": "~0.32.1",
+		"@rocket.chat/ui-kit": "workspace:~",
```

Comment:
I particularly go for `yarn add --tilde` when adding new dependency because "tilde-dependencies" deny minor-version bumps, while carets not; in my personal projects, I even put `defaultSemverRangePrefix: "~"` on `.yarnrc.yml`.
When the package is published on npmjs registry, `workspace:~` is replaced by `~x.y.z` while `workspace:^` is replaced by `^x.y.z`.

Reply:
> When the package is published on npmjs registry, `workspace:~` is replaced with `~x.y.z` while `workspace:^` is replaced with `^x.y.z`.

not really, we always set to `^x.y.z` :
https://github.com/RocketChat/Rocket.Chat/blob/05f45dcc580dd7ee58ce1a91e7c5098c8bfd6f08/packages/release-action/src/fixWorkspaceVersionsBeforePublish.ts#L49

Reply:
Ideally we shouldn't have to do this heavy-lifting ourselves... https://yarnpkg.com/features/workspaces#cross-references

Reply:
"Na prática, a teoria é outra."

Reply:
Patched the action's code.